frameworks/remix-run/application/src/routes/$langstore.shop.$folder_.$product.tsx

```diff
@@ -33,6 +43,8 @@ export const loader: LoaderFunction = async ({ request, params }) => {
 };
 
 export default () => {
-    const { data } = useLoaderData();
+    const { data } = useLoaderData() as {
```

Comment:
`useLoaderData<typeof loader>();` + change the loader using `LoaderFunctionArgs`